Source/WebCore/rendering/updating/RenderTreeUpdater.cpp

```diff
@@ -759,6 +760,15 @@ static std::optional<DidRepaintAndMarkContainingBlock> repaintAndMarkContainingB
         markContainingBlockDirty(*destroyRootRenderer);
     }
 
+    for (auto it = composedTreeDescendantsIterator.begin(), end = composedTreeDescendantsIterator.end(); it != end; ++it) {
```

Comment:
I don't think we should add a new for loop.
We should just do this in the for loop below.